### DISCUSSION TITLE
add additional entity for power consumption

### DISCUSSION THREAD

**Discussion by DrJon85** [klaasnicolaas/home-assistant-glow#184] [Q&A] [answered]
is it possible to add an additional entity from the code? currently in normal setup you are given House - Daily Energy and House - Total Energy (marked with red) my question is is it possible to add the House - Power Consumption (marked with green) or indeed any of the others?
my usage is high as i have the washing machine on!

![Web capture_27-7-2022_13436_10 27 30 25](https://user-images.githubusercontent.com/57183034/181242661-c057489d-3aed-425d-ab42-269933b608f3.jpeg)

**Comment by klaasnicolaas** [comment, marked as answer]
![Screenshot_20220727-144845_Home Assistant.jpg](https://user-images.githubusercontent.com/20448157/181250574-61380292-27d5-4271-9a88-a8dbed4c4376.jpg)

By default you should also have the entity for power consumption 🤷‍♂️
